test/Bugs/profiledArgs.js

```diff
@@ -0,0 +1,12 @@
+//-------------------------------------------------------------------------------------------------------
+// Copyright (C) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE.txt file in the project root for full license information.
+//-------------------------------------------------------------------------------------------------------
+
+function bar(){}

```

Comment:
nit: you can add this test in misc_bugs.js, so that you can have better description and no test file to add.